### PR TITLE
Improve doc note about configuration from brew

### PR DESCRIPTION
**Pull request by HartS**
When installed from homebrew, custom.ini isn't read. Instead, the grafana.ini file [used by the brew service](https://github.com/Homebrew/homebrew-core/blob/9520636eb9fc9d434b533645ef1c96eebd2cc43c/Formula/grafana.rb#L54) will need to be edited directly.

### CODE REVIEW

**Comment by oddlittlebird** [comment]
@torkelo or @marefr or @ryantxu , is this behavior a bug? If so, then we should fix the code rather than make this change to the docs.

**Comment by osg-grafana** [comment]
What OS does this relate to? MacOS, Linux, or both?

**Comment by achatterjee-grafana** [comment]
Need developer review. I am not sure if configuring the *.ini file is a good idea.

**Comment by HartS** [comment]
@osg-grafana osX big sur:

```
$ sw_vers
ProductName:	macOS
ProductVersion:	11.2
BuildVersion:	20D64
```

It hadn't occurred to me that this might work differently on Linux. The line I linked specifies `grafana.ini` as the config file. Does grafana attempt to load a `custom.ini` internally? If so, what path should it attempt to find the file in? I put `custom.ini` in several places before changing `grafana.ini` directly in `/usr/local/etc/grafana/grafana.ini`

- `/usr/local/etc/grafana/custom.ini`
- `/usr/local/share/grafana/conf/custom.ini`

I believe `HOMEBREW_PREFIX` in the formula is just resolving to `/usr/local` on my system, though `/usr/local/share/grafana` is a symlink to `/usr/local/Cellar/grafana/7.5.1/share/grafana/`

**Comment by HartS** [comment]
> Need developer review. I am not sure if configuring the *.ini file is a good idea.

Configuring `grafana.ini` is what I needed to get things working for me, otherwise it uses the default port 3000, which actually *does* work, but shadows my development server connecting to the same data source. MacOS allowing new services to bind to the same port existing services have been using is a bit surprising to me, but that's a different story...

The grafana.ini file in question enumerates all possible settings, all of which are commented out with a preceding `;` (which the doc section immediately following my change notes as the standard way to comment .ini files). I haven't found a `grafana.ini` file (or any `.ini` file for that matter) with the default grafana settings applied; I assume grafana is applying its own default settings for when configuration isn't found in a `.ini` file

**Comment by achatterjee-grafana** [comment]
> > Need developer review. I am not sure if configuring the *.ini file is a good idea.
> 
> Configuring `grafana.ini` is what I needed to get things working for me, otherwise it uses the default port 3000, which actually _does_ work, but shadows my development server connecting to the same data source. MacOS allowing new services to bind to the same port existing services have been using is a bit surprising to me, but that's a different story...
> 
> The grafana.ini file in question enumerates all possible settings, all of which are commented out with a preceding `;` (which the doc section immediately following my change notes as the standard way to comment .ini files). I haven't found a `grafana.ini` file (or any `.ini` file for that matter) with the default grafana settings applied; I assume grafana is applying its own default settings for when configuration isn't found in a `.ini` file

Thanks for the reply and we appreciate your contribution! Let's wait for the dev team to chime in, then we will move forward.

**Comment by HartS** [comment]
I understand that you don't want to document a behaviour that doesn't align with intended behaviour of Grafana and related packages. However, this is the behaviour of the current packages, and while waiting for the dev team to triage, schedule, and release a fix (which I know can take some time, as I imagine you have a considerable backlog of low-priority issues which tends to accumulate in large software projects) the docs are going to be incorrect without this merge.

I'd be happy to file a separate issue to be used for tracking progress of fixing the the homebrew packages to use custom.ini, and (after merging this) another doc issue for changing docs back, blocked on that issue, if you think it would help ensure that item doesn't get lost, while still enabling docs to accurately document current behaviour which could take some time to change based on how things move.

**Comment by achatterjee-grafana** [comment]
@marefr and @aknuds1, can you review this issue when you have a moment. Thank you!

**Review by marefr — APPROVED** [review]
LGTM based on how the current homebrew recipe is implemented

File: docs/sources/administration/configuration.md
Comment (achatterjee-grafana) [inline docs/sources/administration/configuration.md:39]:
```suggestion
By default, the configuration file is located at `/usr/local/etc/grafana/grafana.ini`. For a Grafana instance that was installed with Homebrew, edit the `grafana.ini` file directly. Otherwise, add a configuration file named `custom.ini` to the `conf` folder to override any of the settings defined in `conf/defaults.ini`.
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Added copy edit suggestions.

**Comment by achatterjee-grafana** [comment]
Thanks, @marefr for the review. @HartS, I have added some copy-edit suggestions. As soon as you accept them, we can merge your PR. Thanks again for the contribution!

**Review by achatterjee-grafana — APPROVED** [review]

**Comment by oddlittlebird** [comment]
@HartS, please update from master and push the changes. Hopefully that will fix the failing test.

**Comment by HartS** [comment]
Thanks @oddlittlebird , I just went ahead and did that

**Comment by HartS** [comment]
@oddlittlebird FYI the `build-docs-website` drone step has been failing in the other merged docs changes from today also. I tried to follow the trail of 'what changed'  but it ends at having no visibility into how https://hub.docker.com/r/grafana/docs-base is built and therefore, not knowing what `make prod` is supposed to do

**Comment by achatterjee-grafana** [comment]
Thanks, for your patience @HartS. We are aware and are investigating.

**Comment by oddlittlebird** [comment]
The fix has been pushed to master. Please update your branch again, and this will hopefully be merged. Thanks for contributing!

**Comment by HartS** [comment]
Thanks for helping me get it merged @oddlittlebird! I just merged in master to my branch again.

**Pull request merged**
